velox/expression/tests/CastExprTest.cpp

```diff
@@ -14,12 +14,17 @@
  * limitations under the License.
  */
 
+#include <cstdint>
 #include <limits>
+
 #include "velox/buffer/Buffer.h"
 #include "velox/common/memory/Memory.h"
+#include "velox/core/Expressions.h"
+#include "velox/core/ITypedExpr.h"
 #include "velox/expression/ControlExpr.h"
 #include "velox/expression/VectorFunction.h"
 #include "velox/functions/prestosql/tests/FunctionBaseTest.h"
+#include "velox/functions/prestosql/types/JsonType.h"
```

Comment:
code engine tests should not depend on application-specific code, e.g. there should be no dependency on custom types; rather, the test for JSON cast should go into velox/functions/prestosql/tests

Reply:
Thank you for pointing this out! I moved the tests of casting to Json into velox/functions/prestosql/tests/CustomTypeCastTest.cpp.